test/images/pets/zookeeper-installer/on-start.sh

```diff
@@ -94,7 +94,7 @@ ADD_SERVER="server.$MY_ID=$MY_NAME:2888:3888:participant;0.0.0.0:2181"
 
 # Prove that we've actually joined the running cluster
 ITERATION=0
-until $(echo config | /opt/nc localhost 2181 | grep "${ADD_SERVER}" > /dev/null); do
+until echo config | /opt/nc localhost 2181 | grep "${ADD_SERVER}" > /dev/null; do
```

Comment:
Why I do this change:

```
In test/images/pets/zookeeper-installer/on-start.sh line 97:
until $(echo config | /opt/nc localhost 2181 | grep "${ADD_SERVER}" > /dev/null); do
      ^-- SC2091: Remove surrounding $() to avoid executing output.
```